tools/mage/dev.go

```diff
@@ -151,6 +151,16 @@ func (Dev) DBRestore(ctx context.Context) error {
 	).Error
 }
 
+// RedisFlush deletes all keys from redis and restore the database to its initial state.
+func (Dev) RedisFlush() error {
+	if mg.Verbose() {
+		fmt.Println("Deleting all keys from redis")
+	}
+
+	_, err := sh.Exec(nil, nil, os.Stderr, "docker-compose", dockerComposeFlags("exec", "-T", "redis", "redis-cli", "FLUSHDB")...)
```

Comment:
Honestly I'm not a fan of this and I think we should only be removing keys we need to be removing here (i.e. delete in a namespace).

Reply:
E.g. that's how we do it when testing https://github.com/TheThingsNetwork/lorawan-stack/blob/1282e8fc385341703db4e3fa18fc593af593da8c/pkg/util/test/redis.go#L121-L135